package.json

```diff
@@ -35,7 +35,7 @@
     "/ios",
     "/macos",
     "/schema.json",
-    "/scripts",
+    "/scripts/*.js",
```

Comment:
ok so the assumption/rule here is that we are expecting to have all the .js scripts to be "external" and the *.mjs as internal only, right?

I think it's ok, but maybe document it in the contribution docs? I fear we'll forget unless we have some constant reminder somewhere

Reply:
That's a good callout. I've updated the contribution docs. Hopefully, our integration tests will also catch missing files.